output/schema/schema.json

```diff
@@ -25433,9 +25445,60 @@
               "namespace": "_types"
             }
           }
+        },
+        {
+          "description": "Boost value to apply to kNN scores",
+          "name": "boost",
+          "required": false,
+          "type": {
+            "kind": "instance_of",
+            "type": {
+              "name": "float",
+              "namespace": "_types"
+            }
+          }
+        },
+        {
+          "description": "Filters for the kNN search query",
+          "name": "filter",
+          "required": false,
+          "type": {
+            "items": [
+              {
+                "kind": "instance_of",
+                "type": {
+                  "name": "QueryContainer",
+                  "namespace": "_types.query_dsl"
+                }
+              },
+              {
+                "kind": "array_of",
+                "value": {
+                  "kind": "instance_of",
+                  "type": {
+                    "name": "QueryContainer",
+                    "namespace": "_types.query_dsl"
+                  }
+                }
+              }
+            ],
+            "kind": "union_of"
+          }
+        },
+        {
+          "description": "The required minimum similarity for a vector to be considered a match",
+          "name": "similarity",
+          "required": true,
```

Comment:
Thanks for the contribution! 

Are we sure `similarity` is a required parameter? I believe it is a requirement for the similarity score when provided but not a required parameter for the knn query, per https://www.elastic.co/guide/en/elasticsearch/reference/8.9/knn-search.html#_search_knn_with_expected_similarity.

Reply:
Yes that was a mistake on my part, thank you for catching that. 

I added a commit that fixes this.

Reply:
Should I click on resolve conversation, or leave this as is for now?